fabio/pilatusimage.py

```diff
@@ -61,7 +61,7 @@ class PilatusImage(tifimage.TifImage):
 
     DEFAULT_EXTENSIONS = ["tif", "tiff"]
 
-    _keyvalue_spliter = re.compile(r"\s*[,:=\s]\s*")
+    _keyvalue_spliter = re.compile(r"\s*[,:=]\s*")
```

Comment:
This contradicts the comment below.

Reply:
Yes. Just i don't see why a space is considered as a separator, but it looks to be the case.
Fortunately we have unittests.